README.md

```diff
@@ -26,105 +26,106 @@
-- `grunt user`: Build a userstyle ready to be pasted on userstyles.org.
-- `grunt usermin`: Same as `grunt user`, but outputs a minified style.
-- `grunt usercss`: Build a [Usercss](https://github.com/openstyles/stylus/wiki/Usercss) style, to be used with modern userstyle engines like [stylus](https://github.com/stylus/stylus).
-- `grunt themes`: Parse all theme files and combine them into minified styles in the `themes` directory.
-- `grunt clean`: Reformat `github-dark.css` to conform the style guide.
-- `grunt lint`: Run Stylelint on `github-dark.css`.
-- `grunt authors`: Regenerate the `AUTHORS` file based on git history.
-- `grunt imagemin`: Minify all images present in the `image` directory.
-- `grunt generate`: Regenerate auto-generated CSS rules based on GitHub's stylesheet. This should be ran regularly.
-- `grunt patch`: Increment the version on the patch version number. Also updates headers and creates a commit.
-- `grunt minor`: Increment the version on the minor version number. Also updates headers and creates a commit.
-- `grunt major`: Increment the version on the major version number. Also updates headers and creates a commit.
-- `grunt update`: Update and install dependencies.
+:eight_spoked_asterisk: `grunt`: The default task will build a custom theme using a provided `build.json` file. See the [wiki](https://github.com/StylishThemes/GitHub-Dark/wiki/Build) for details.
+:eight_spoked_asterisk: `grunt min`: Same as `grunt`, but outputs a minified style.
+:eight_spoked_asterisk: `grunt user`: Build a userstyle ready to be pasted on userstyles.org.
+:eight_spoked_asterisk: `grunt usermin`: Same as `grunt user`, but outputs a minified style.
+:eight_spoked_asterisk: `grunt usercss`: Build a [Usercss](https://github.com/openstyles/stylus/wiki/Usercss) style, to be used with modern userstyle engines like [stylus](https://github.com/stylus/stylus).
+:eight_spoked_asterisk: `grunt themes`: Parse all theme files and combine them into minified styles in the `themes` directory.
+:eight_spoked_asterisk: `grunt clean`: Reformat `github-dark.css` to conform the style guide.
+:eight_spoked_asterisk: `grunt lint`: Run Stylelint on `github-dark.css`.
+:eight_spoked_asterisk: `grunt authors`: Regenerate the `AUTHORS` file based on git history.
+:eight_spoked_asterisk: `grunt imagemin`: Minify all images present in the `image` directory.
+:eight_spoked_asterisk: `grunt generate`: Regenerate auto-generated CSS rules based on GitHub's stylesheet. This should be ran regularly.
+:eight_spoked_asterisk: `grunt patch`: Increment the version on the patch version number. Also updates headers and creates a commit.
+:eight_spoked_asterisk: `grunt minor`: Increment the version on the minor version number. Also updates headers and creates a commit.
+:eight_spoked_asterisk: `grunt major`: Increment the version on the major version number. Also updates headers and creates a commit.
+:eight_spoked_asterisk: `grunt update`: Update and install dependencies.
 
```

Comment:
Breaks the list and looks fugly.

Reply:
Maybe a 🙈 would be more appropriate? LOL

Reply:
:+1:

Reply:
:monkey: it was